contracts/farm-bot.sol

```diff
@@ -357,13 +358,8 @@ contract FarmBot is FarmbotERC20, AccessControl {
             _deadline
         );
 
-        // How much LP we have to re-invest
-        uint256 lpBalance = stakingToken.balanceOf(address(this));
-        stakingToken.approve(address(stakingRewards), lpBalance);
-
-        // Actually reinvest and adjust FP weight
-        stakingRewards.stake(lpBalance);
-        lpTotalBalance += lpBalance;
+        // reinvest LPs and adjust FP weight
+        lpTotalBalance += investInFarm();
```

Comment:
`claimRewards` now no longer has access to information about how much LP was reinvested, right? We need this [in order to update](https://github.com/revo-market/contracts/pull/19/files#diff-bf50f0fc86f5dd1b0182cce33ec6bebd719b56d00ab5427e6596d51d650f68ccR396) the `interestEarnedDenominator` field in your other PR; might be good to wait to merge that PR and resolve merge conflicts in this branch.